CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.30](https://github.com/brave/brave-ios/releases/tag/v1.30)
+
+- Implemented History Sync. ([#3227](https://github.com/brave/brave-ios/issues/3227))
+- Added display ads to the Brave News feed. ([#3872](https://github.com/brave/brave-ios/issues/3872))
+- Added "Delete All" functionality to clear history via history page. ([#3723](https://github.com/brave/brave-ios/issues/3723))
+- Added ability to share selected text from other apps to Brave via share sheet. ([#3863](https://github.com/brave/brave-ios/issues/3863))
+- Fixed app theme being set to light at launch when "Private Browsing" only is being used as default. ([#3591](https://github.com/brave/brave-ios/issues/3591))
```

Comment:
Wrong issue #. Should be `3936`. Probably a copy/paste error when adding the new line.

Reply:
Wrong copy paste via web. Fixed it